### ISSUE TITLE
Show indicator for accounts and folders with unseen messages

### ISSUE THREAD

**Issue by cketti** [k9mail/k-9#680]
See PR #213

**Comment by opapanik** [comment]
Yes, and please add an option to get notified on any new unseen (as opposed to unread) messages. As it stands now, if new 'read' messages arrive at the device, there is no notification.

I understand this has been much discussed and different people want one or the totally opposite thing.

My scenario is, a common email account, a coworker might read new emails from another client before they get to my device. Later, my device downloads new messages I have not seen, but I don't get notified of them because they are marked as read (by my coworker's client).

Also, on the same scenario I (or my coworker), should be notified on any mail sent by the other one (we don't want to 'cc' it to each other). I set up the 'Sent' folder to be polled and push active, but although it gets refreshed with newly sent messages, there is never a notification, even though we made sure they are marked unread on the (other) client used to send them.

**Comment by cketti** [comment]
@opapanik: That is not a use case I want to add support for in K-9 Mail.

**Comment by opapanik** [comment]
Maybe it sounds more complex than it really is. 

All I suggest boils down to an (sub) option at notifications:
"Notify on new unseen messages too"

**Comment by cketti** [comment]
@opapanik: I believe most people don't use email the way you do. So I'd rather spend developer time elsewhere.

**Comment by opapanik** [comment]
Can't argue on that. You 're right, if you think it would be that hard to 
implement it.
